integration-cli/docker_cli_links_test.go

```diff
@@ -141,22 +117,15 @@ func (s *DockerSuite) TestLinksHostsFilesInject(c *check.C) {
 	out, _ = dockerCmd(c, "run", "-itd", "--name", "two", "--link", "one:onetwo", "busybox", "top")
 	idTwo := strings.TrimSpace(out)
 
-	c.Assert(waitRun(idTwo), check.IsNil)
+	c.Assert(waitRun(idTwo), checker.IsNil)
 
 	contentOne, err := readContainerFileWithExec(idOne, "/etc/hosts")
-	if err != nil {
-		c.Fatal(err, string(contentOne))
-	}
+	c.Assert(err, checker.IsNil, check.Commentf("contentOne: %s", string(contentOne)))
 
 	contentTwo, err := readContainerFileWithExec(idTwo, "/etc/hosts")
-	if err != nil {
-		c.Fatal(err, string(contentTwo))
-	}
-
-	if !strings.Contains(string(contentTwo), "onetwo") {
-		c.Fatal("Host is not present in updated hosts file", string(contentTwo))
-	}
-
+	c.Assert(err, checker.IsNil, check.Commentf("contentTwo: %s", string(contentTwo)))
+	// Host is not present in updated hosts file
+	c.Assert(string(contentTwo), checker.Contains, "onetwo")
```

Comment:
```	
// Host is not present in updated hosts file
c.Assert(string(contentTwo), checker.Contains, "onetwo")
```
This won't lose the "context", actually "Host is not present in updated hosts file" will also be printed out when the assertion fails since `c.Assert` can handle annotation above it as my little test shows.
So that's why I put the context above the `c.Assert` statement instead of putting it in `check.Commentf` or in the end of the line, that won't make sentence too long to read.
Will this be acceptable? :smile: